ee-9/feature-pack/pom.xml

```diff
@@ -90,7 +90,7 @@
         <version.org.hibernate.validator>7.0.1.Final</version.org.hibernate.validator>
         <version.org.jberet>2.0.1.Final</version.org.jberet>
         <version.org.jboss.activemq.artemis.integration>1.0.4</version.org.jboss.activemq.artemis.integration>
-        <version.org.jboss.metadata>14.0.0.Beta2</version.org.jboss.metadata>
+        <version.org.jboss.metadata>14.0.0.Final</version.org.jboss.metadata>
```

Comment:
How about dropping this property, changing the version in the root pom, and then in this pom remove any jb-metadata dependency info except for the two -jakarta artifacts? In the long run I don't see why regular WF, even an EE 8 based variant, needs to stay on JBM 13.

Reply:
ok